Feature/clear value

RW-KFlerlage opened the pull request:
This pull aims to fix issue #82. By adding a clear function inside of the component we can lift it out by the useRefs hook and allowing us to clear the input box with a button or whenever. Also added a section to the ReadMe explaining how to do this.

Closes #82

Tintef commented on README.md:
```diff
@@ -464,3 +464,20 @@ result = {
+        ref={googlePlacesAutocompleteRef}
+        onSelect={(result) => console.log(result)}
+      />
+      <button onClick={() => googlePlacesAutocompleteRef.current?.clearValue()}>Clear Button</button>
```
Although this is correct, using conditional chaining is not available on es6. Can you update it?

RW-KFlerlage commented:
I updated the docs to remove the conditional changing, It ended up I didn't need it. I also removed the onSelect and added importing useRefs to match the rest of the docs.
